docs/source/introduction.md

```diff
@@ -279,6 +279,24 @@ using `zip://..`:
 >>> read_dataframe("zip://ne_10m_admin_0_countries.zip")
 ```
 
+If your archive contains multiple datasets, you need to specify which one to use;
+otherwise GDAL will default to the first one found.
+
+```python
+>>> read_dataframe("/vsizip/ne_10m_admin_0_countries.zip/a/b/test.shp")
```

Comment:
Maybe use a dummy name instead of "ne_10m_admin_0_countries.zip", because this name denotes a specific naturalearth dataset, for which it would be a bit strange to have multiple shapefiles